www/src/components/homepage/homepage-newsletter.js

```diff
@@ -1,110 +1,98 @@
-import React from "react"
-import styled from "@emotion/styled"
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { Flex } from "theme-ui"
```

Comment:
can we club this with the jsx import above it?

Reply:
👍